### ISSUE TITLE
Add download images to merged download file docs

### ISSUE THREAD

**Issue by sjspielman** [AlexsLemonade/scpca-docs#245]
I think as long as we file an issue for the images and note the changes that we are waiting on, then this can wait for another PR.

_Originally posted by @allyhawkins in https://github.com/AlexsLemonade/scpca-docs/pull/234#discussion_r1459638718_

---

We are currently waiting for updating download images for merged docs. Once the images have been made, we'll need to add them to the the merged download docs.

**Comment by sjspielman** [comment]
closed by #256
